GrandCentralBoard/Core/Configuration/WidgetSettings.swift

```diff
@@ -22,6 +22,36 @@ public struct WidgetSettings {
     }
 }
 
+extension WidgetSettings : Hashable {
+    public var hashValue: Int {
+
+        if let settings = settings as? [String : AnyObject] {
+            return settings.enumerate().reduce(name.hashValue) { first, second in
+
+                if let string = second.element.1 as? String {
```

Comment:
I think it would be more concise using:
```
switch second.element.1 {
case let number as NSNumber:
    return first ^ second.element.0.hashValue ^ string.hashValue
case let string as String:
    return first ^ second.element.0.hashValue ^ string.hashValue
case let null as NSNull:
     return first ^ second.element.0.hashValue
default:
    return first
}
```
Or better yet enforce `setting` in `WidgetSettings` to be `Hashable`